<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DEEP-1705211512" ignored="1" name="test1" tests="2" failures="1" timestamp="04 十二月 2017 00:16:38 GMT" time="0.028" errors="0">
  <testcase name="testPrintMessage" time="0.009" classname="TestNG.GroupTestExample"/>
  <testcase name="testingExitMessage" time="0.001" classname="TestNG.GroupTestExample">
    <failure type="java.lang.AssertionError" message="expected [www..com] but found [wwwtutorialspoit.com]">
      <![CDATA[java.lang.AssertionError: expected [www..com] but found [wwwtutorialspoit.com]
	at TestNG.GroupTestExample.testingExitMessage(GroupTestExample.java:32)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testingExitMessage -->
  <testcase name="testSalutationMessage" time="0.0" classname="TestNG.GroupTestExample">
    <ignored/>
  </testcase> <!-- testSalutationMessage -->
</testsuite> <!-- test1 -->
